### PR TITLE
[3.0.8.0] Added ledger support for claim tool

### PR DESCRIPTION
**Pull request by fenix2222**
This enables ability to extract and sign eligible addresses from ledger wallet for STRAX distribution.

### CODE REVIEW

File: src/AddressOwnershipTool.Tests/BlockExplorerTests.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    public class BlockExplorerClientTests
+    {
+        [Fact]
+        public void GivenAddressWIthFunds_WhenCheckingIfHasBalance_ThenTrueIsReturned()
+        {
+            var address = "Sdm4cmKhBtKehvXkpRSVaTX1RAjsE5CVNc";
+
+            var blockExplorerClient = new BlockExplorerClient();
+
+            var result = blockExplorerClient.HasBalance(address);
```
Comment (zeptin) [inline src/AddressOwnershipTool.Tests/BlockExplorerTests.cs:14]: I am not sure that tying this test to the existence of an externally-hosted API is a good idea

File: src/AddressOwnershipTool/BlockExplorerClient.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    public class BlockExplorerClient
+    {
+        private const string ExplorerBaseUrl = "https://stratissnapshotapi.stratisplatform.com/";
+
+        public bool HasBalance(string address)
+        {
+            var stratisApiClient = new NodeApiClient($"{ExplorerBaseUrl}api");
+            var balance = stratisApiClient.GetAddressBalance(address);
+
+            Console.WriteLine($"Balance for {address} is {(balance / 100_000_000):N8}");
```
Comment (zeptin) [inline src/AddressOwnershipTool/BlockExplorerClient.cs:14]: This line can be removed

File: src/AddressOwnershipTool/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (zeptin) [inline src/AddressOwnershipTool/Properties/launchSettings.json:1]: This file can be removed

File: src/LedgerWallet/packages.config [hunk 4]
```diff
@@ -0,0 +1,38 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="hidlibrary" version="3.2.31.0" targetFramework="net45" />
+  <package id="Microsoft.NETCore.Platforms" version="1.1.0" targetFramework="net45" />
+  <package id="NStratis" version="4.0.0.77" targetFramework="net45" />
```
Comment (zeptin) [inline src/LedgerWallet/packages.config:5]: This should reference the NBitcoin project in the solution instead

File: src/AddressOwnershipTool.Tests/BlockExplorerTests.cs (hunk 1 above)
Comment (fenix2222) [reply src/AddressOwnershipTool.Tests/BlockExplorerTests.cs:14]: Removed

File: src/AddressOwnershipTool/BlockExplorerClient.cs (hunk 2 above)
Comment (fenix2222) [reply src/AddressOwnershipTool/BlockExplorerClient.cs]: Done

File: src/AddressOwnershipTool/Properties/launchSettings.json (hunk 3 above)
Comment (fenix2222) [reply src/AddressOwnershipTool/Properties/launchSettings.json]: Done

File: src/LedgerWallet/packages.config (hunk 4 above)
Comment (fenix2222) [reply src/LedgerWallet/packages.config:5]: Removed

**Review by zeptin — APPROVED** [review]

**Review by zeptin — APPROVED** [review]

**Review by zeptin — APPROVED** [review]

**Pull request merged**